components/Account.tsx

```diff
@@ -83,23 +92,63 @@ const Account = ({ triedToEagerConnect }: Props) => {
   }
 
   return (
-    <a style={{"padding":"20px"}}
-      {...{
-        href: formatEtherscanLink("Account", [chainId, account]),
-        target: "_blank",
-        rel: "noopener noreferrer",
-      }}
-    >
-      {ENSName || account}   
-            <p style={{"fontSize": "80%"}}
-            ><i> --- Rinkeby Only! --- </i></p>
-            <a
-              href="https://rinkeby.etherscan.io/address/0x11398bf5967Cd37BC2482e0f4E111cb93D230B05#readContract"
-            >
-              <u>Contract on Etherscan</u>
-            </a>
-    </a>
+    <div style={{"padding":"20px", "marginLeft": "auto", "marginRight": "auto", "marginBottom": "1em", "maxHeight": "49em", "overflow": "scroll"}}>
+      <WizardList wizardsContract={wizardsContract} account={account} storageContract={storageContract} verified={verified} wizardTraits={wizardTraits}/>   
+    </div>
   );
 };
 
+
+const WizardGrid = ({wizards, verified, wizardTraits}) => {
+  const setWizard = useStore(state => state.setWizard);
```

Comment:
```suggestion
  const setWizard = useStore(state => setWizard);
```
for some reason otherwise this fails